.github/workflows/service_test_ghac.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Test
         shell: bash
         working-directory: core
-        run: cargo nextest run behavior --features tests,services-ghac
+        run: cargo test behavior --features tests,services-ghac
```

Comment:
May I ask what's the rationale here?

Reply:
Nextest relies on `path/to/test/binary list` to enumerate all test cases. If any logs are output at this stage, it will refuse to run.

Output likes:

```shell
error: creating test list failed

Caused by:
  for `opendal::behavior`, line '  2025-02-14T11:57:30.185063Z DEBUG opendal::services: service=ghac name=opendal: metadata started' did not end with the string ': test' or ': benchmark'
full output:
behavior::test_read_full: test
behavior::test_read_range: test
behavior::test_reader: test
behavior::test_reader_with_if_match: test
behavior::test_reader_with_if_none_match: test
```